<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CensoArgentinoDialog</class>
 <widget class="QDialog" name="CensoArgentinoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Censo Argentino</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>

     <!-- BROWSE TAB -->
     <widget class="QWidget" name="tabBrowse">
      <attribute name="title">
       <string>Explorar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Año:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboYear"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nivel Geográfico:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboGeoLevel"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Filtrar por Geografía (opcional):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listGeoFilter">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tipo de Entidad:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboEntityType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Variables (marcar múltiples):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchVariables">
         <property name="placeholderText">
          <string>Buscar variables...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnSelectAllVars">
           <property name="text">
            <string>Seleccionar Todo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearAllVars">
           <property name="text">
            <string>Limpiar Selección</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listVariables">
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDescription">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStatus">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFilterViewbox">
         <property name="text">
          <string>Filtrar por extensión actual del mapa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Cargar Capa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSaveNotice">
         <property name="text">
          <string>&lt;i&gt;Nota: Las capas son temporales. Para guardar, haga clic derecho en la capa en QGIS y seleccione "Hacer permanente".&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <!-- SQL TAB -->
     <widget class="QWidget" name="tabSql">
      <attribute name="title">
       <string>SQL</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblSqlHelp">
         <property name="text">
          <string>Tablas disponibles: &lt;b&gt;radios&lt;/b&gt;, &lt;b&gt;census&lt;/b&gt;, &lt;b&gt;metadata&lt;/b&gt;&lt;br/&gt;Incluya &lt;code&gt;ST_AsText(g.geometry) as wkt&lt;/code&gt; para cargar resultados como capa de mapa</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtSql">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas,Monaco,Courier New,monospace</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboExamples">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnRunSql">
           <property name="text">
            <string>Ejecutar Consulta</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBarSql">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSqlStatus">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblSqlResults">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <!-- QUERY LOG TAB -->
     <widget class="QWidget" name="tabQueryLog">
      <attribute name="title">
       <string>Registro de Consultas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lblQueryLogHelp">
         <property name="text">
          <string>Las consultas SQL generadas desde la pestaña Explorar y las consultas manuales SQL aparecen aquí</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtQueryLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Consolas,Monaco,Courier New,monospace</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnCopyQuery">
           <property name="text">
            <string>Copiar Última Consulta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearLog">
           <property name="text">
            <string>Limpiar Registro</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>

    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
